oot.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/bash
+
+echo $#
+
+if (( "$#" != 1 )); then
+	echo 'Usage: ./oot/oot.sh <device> (Example for akatsuki: ./oot/oot.sh akatsuki)'
+	exit
+fi
+
+_device="$1"
+
+
+# Device specific
+case ${_device} in
+discovery|pioneer)
+	_platform=nile
+	;;
+mermaid)
+	_platform=ganges
+	;;
+akatsuki)
+	_platform=tama
+	;;
+bahamut|griffin)
+	_platform=kumano
+	;;
+*)
+	echo "Device '${_device}' unknown or not implemented"
+	exit
+	;;
+esac
+
+
+# Platform common
+BOARD_KERNEL_BASE=0x00000000
+BOARD_KERNEL_PAGESIZE=4096
+BOARD_KERNEL_TAGS_OFFSET=0x01E00000
+BOARD_RAMDISK_OFFSET=0x02000000
+
+BOARD_KERNEL_CMDLINE="${BOARD_KERNEL_CMDLINE} lpm_levels.sleep_disabled=1"
+BOARD_KERNEL_CMDLINE="${BOARD_KERNEL_CMDLINE} service_locator.enable=1"
```

Comment:
What do you reckon, shall we replace these and the below with `+=" ..."`?

```suggestion
BOARD_KERNEL_CMDLINE+=" lpm_levels.sleep_disabled=1"
BOARD_KERNEL_CMDLINE+=" service_locator.enable=1"
```